Rubrik/Public/Get-RubrikAPIToken.ps1

```diff
@@ -0,0 +1,79 @@
+#requires -Version 3
+function Get-RubrikAPIToken
+{
+  <#  
+      .SYNOPSIS
+      Connects to Rubrik and retrieves a list of generated API tokens
+
+      .DESCRIPTION
+      The Get-RubrikAPIToken cmdlet is used to pull a list of generated API tokens from the Rubrik cluster.
+
+      .NOTES
+      Written by Mike Preston for community usage
+      Twitter: @mwpreston
+      GitHub: mwpreston
+
+
+      .LINK
+     http://rubrikinc.github.io/rubrik-sdk-for-powershell/reference/Get-RubrikAPIToken.html
+
+      .EXAMPLE
+      Get-RubrikAPIToken
+      This will return all generated API tokens belonging to the currently logged in user.
+
+      Get-RubrikAPIToken -tag roxie
+      This will return all generated API tokens belonging to the currently logged in user with a 'roxie' tag.
+
+      Get-RubrikAPIToken -organizationId 1111-2222-3333
+      This will return all generated API tokens assigned to the currently logged in user with the specified organization id.
+  #>
+
+  [CmdletBinding()]
+  Param(
+    # UserID to retrieve tokens from - defaults to currently logged in user
+    [Alias('user_id')]
+    [String]$UserId = $rubrikconnection.userId,
+    # Tag assigned to the API Token
+    [String]$Tag,
+    [String]$OrganizationId,
```

Comment:
Missing comment for $OrganizationId param

Reply:
Fixed